INSTALL.rst

```diff
@@ -0,0 +1,117 @@
+Installing the GA4GH Schemas
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+The schemas are documents (text files) that formally describe the
+messages that pass between GA4GH reference servers and clients, which we
+also refer to collectively as "the API." The schemas are written in a
+language called `Avro <http://avro.apache.org>`__.
+
+We use the schemas in a couple of different ways: - to generate source
```

Comment:
These were bullets in MD:

https://github.com/reece/schemas/blob/documentation/INSTALL.rst